api: allow forcing on-device processing (#1262)
imputnet/cobalt: docs/api-env-variables.md

this document is not final and will expand over time. feel free to improve it!

### general vars
| name                   | default   | value example                         |
|:-----------------------|:----------|:--------------------------------------|
| API_URL                |           | `https://api.url.example/`            |
| API_PORT               | `9000`    | `1337`                                |
| COOKIE_PATH            |           | `/cookies.json`                       |
| PROCESSING_PRIORITY    |           | `10`                                  |
| API_INSTANCE_COUNT     |           | `6`                                   |
| API_REDIS_URL          |           | `redis://localhost:6379`              |
| DISABLED_SERVICES      |           | `bilibili,youtube`                    |

> | FORCE_LOCAL_PROCESSING |           | `always`                              |

Reviewer: ```suggestion
| FORCE_LOCAL_PROCESSING | `never`   | `always`                              |
```

the value is a string of cobalt-supported services.

### FORCE_LOCAL_PROCESSING

> the value is a string, either `always` or `session`.

Reviewer: ```suggestion
the value is one of `never` (default), `session` or `always`.
```
